object_frame_station[] station_objs
object_frame[] objs